coops-ui/src/mixins/address-mixin.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Vue } from 'vue-property-decorator'
+import isEqual from 'lodash.isequal'
+
+/**
+ * Mixin that provides some useful address utilities.
+ */
+@Component
+export default class AddressMixin extends Vue {
+  /**
+   * Compares two address objects and returns a boolean indicating
+   * if they match.
+   *
+   * @param addressA An address object.
+   * @param addressB An address object.
+   * @return boolean True if they match, false if they do not.
+   */
+  isSameAddress (addressA: Object, addressB: Object): boolean {
```

Comment:
I thought about this and I like it!

But now we have a second way to compare addresses (see sameAddress() in RegisteredOfficeAddress.vue). (We also compare directors that way, but I'm undecided which way is best.)

Can you decide which way we should use to compare addresses and update the code to use the same mechanism everywhere?

Reply:
Absolutely, i will look into the other implementation and we can discuss offline!

Reply:
Hmm, i like the other method but using a library like Lodash really abstracts the complexit. Also would have to agree, i like creating this sort of utility to use App wide opposed to local implementation in a specific component.  Would you prefer to see this refactor done in this ticket/pr, or should that be stand alone in another small ticket?

Reply:
No preference on making the change here or in a new ticket. I can see pros and cons to both.

Reply:
Ok, sounds good. Might as well do it now, will update and do some testing 👍

Reply:
After our discussion this afternoon, i've decided to leave the refactor of the existing code for a new ticket due to there being some more complexity and more thorough testing. If you are ok with that we can move this forward and refactor the `other` comparison in a new ticket?

Reply:
Also, in that new ticket, please request to delete the addressType when fetching office address (in App.vue) and also in the unit tests that reference it.

Once the new ticket is created, please resolve this conversation. Thx.

Reply:
To follow up on this, this refactor work has been done in the following PR